src/pe_reports/pages.py

```diff
@@ -14,27 +15,35 @@
 class Pages:
     """Class containing the presentation page information."""
 
+    @staticmethod
     def cover(prs):
         """Page 1: Cover page of presentation."""
         slide = prs.slides[0]
         shape = Paragraph.shapes(prs, slide)
         text_frame = Paragraph.text_frame(prs, shape)
         frame = text_frame.paragraphs[0]
         run = frame.add_run()
-        df_customer = pd.read_csv("src/pe_reports/data/csv/dhs_customer.csv")
-        dates = (
-            df_customer.iloc[0]["start_date"] + " to " + df_customer.iloc[0]["end_date"]
-        )
-        run.text = (
-            "Prepared for: "
-            + str(df_customer.iloc[0]["name"])
-            + "\nReporting period: "
-            + str(dates)
-        )
-        font = run.font
-        font = Paragraph.text_style_title(prs, font)
+        try:
+            df_customer = pd.read_csv("src/pe_reports/data/csv/dhs_customer.csv")
+            dates = (
+                df_customer.iloc[0]["start_date"]
+                + " to "
+                + df_customer.iloc[0]["end_date"]
+            )
+            run.text = (
+                "Prepared for: "
+                + str(df_customer.iloc[0]["name"])
+                + "\nReporting period: "
+                + str(dates)
+            )
+            font = run.font
+            Paragraph.text_style_title(prs, font)
+        except Exception:
```

Comment:
We should catch something more specific than `Exception` here and in all similar cases below.

Reply:
Resolved in 675a8e4